src/autowiring/CMakeLists.txt

```diff
@@ -244,7 +245,7 @@ target_include_directories(
   "${PROJECT_SOURCE_DIR}/contrib/autoboost"
   PUBLIC
   "$<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/src>"
-  "$<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}>"
+  "$<BUILD_INTERFACE:${PROJECT_BUILD_DIR}/src>"
```

Comment:
It's passing on Travis though . I have tested both in-source and out-of-source builds.

What are the steps to reproduce the error you're seeing?